cmd/helpers.go

```diff
@@ -52,14 +58,50 @@ func unmarshalConfigFile() (*config.Config, error) {
 		return nil, errors.Wrapf(err, "Error reading config from file<%s>", cfgFile)
 	}
 
+	var overrides []string
+	cfgJSON, overrides, err = overrideScriptValues(cfgJSON)
+	if err != nil {
+		return nil, errors.WithStack(err)
+	}
+
 	var cfg config.Config
 	if err = jsonit.Unmarshal(cfgJSON, &cfg); err != nil {
 		return nil, errors.Wrap(err, "Failed to unmarshal config from json")
 	}
 
+	if cfg.Settings.LogSettings.Format != config.LogFormatNoLogs {
+		PrintOverrides(overrides)
+	}
+
 	return &cfg, nil
 }
 
+func overrideScriptValues(cfgJSON []byte) ([]byte, []string, error) {
+	overrides := make([]string, 0, len(scriptOverrides))
+	for _, kvp := range scriptOverrides {
+		kvSplit := strings.SplitN(kvp, "=", 2)
+		if len(kvSplit) != 2 {
+			return cfgJSON, overrides, errors.Errorf("malformed override: %s, should be in the form key.path=value", kvp)
+		}
+		path := helpers.DataPath(kvSplit[0])
+		rawOrig, err := path.Lookup(cfgJSON)
```

Comment:
Still not entirely sure about this one 🤔  Also if the override is a json object, the individual keys won't be checked. I do understand it's a good thing to throw errors whenever possible, though, so removing it outright is not the best option either.

Reply:
Not an easy one. My thinking, right or not, is that mistakes will more likely be done when writing the path, if an entire object is used it's at least more likely to be copy-pasted. My vote is on keeping the check, but agree not optimal, don't have a better solution.

Reply:
Yeah I tend to agree if the choice is between keeping it and removing it outright.